deployer/deployer-services/src/main/java/io/hyscale/deployer/services/handler/impl/V1StatefulSetHandler.java

```diff
@@ -392,15 +392,16 @@ public String getPodSelector(ApiClient apiClient, String selector, boolean label
         if (stsStatus == null) {
             return selector;
         }
-
-        String currentRevision = stsStatus.getCurrentRevision();
-        String updateRevision = stsStatus.getUpdateRevision();
-        logger.debug("Current Revision = " + currentRevision);
-        logger.debug("Updated Revision = " + updateRevision);
-
-        return selector
-                .concat("," + K8SRuntimeConstants.K8s_STS_CONTROLLER_REVISION_HASH + "=" + updateRevision);
+        return getPodSelector(stsStatus, selector);
     }
+    
+	public String getPodSelector(V1StatefulSetStatus stsStatus, String selector) {
```

Comment:
Why is it decoupled ? 
The merge of labels should a new abstract method ! Think on this